package.json

```diff
@@ -40,21 +40,28 @@
     "encodeurl": "~2.0.0",
     "escape-html": "~1.0.3",
     "etag": "~1.8.1",
+    "express": "^4.19.2",
+    "express-rate-limit": "^7.2.0",
     "finalhandler": "1.2.0",
     "fresh": "0.5.2",
+    "helmet": "^7.1.0",
     "http-errors": "2.0.0",
+    "i18n": "^0.15.1",
     "merge-descriptors": "1.0.1",
     "methods": "~1.1.2",
     "on-finished": "2.4.1",
     "parseurl": "~1.3.3",
+    "path": "^0.12.7",
     "path-to-regexp": "0.1.7",
     "proxy-addr": "~2.0.7",
+    "qrcode": "^1.5.3",
```

Comment:
This dependency requires Node@10 at least: https://github.com/soldair/node-qrcode/blob/master/package.json#L68. Currently we support 0.10 the minimum: https://github.com/expressjs/express/blob/master/.github/workflows/ci.yml#L29